build.gradle

```diff
@@ -386,18 +377,13 @@ dependencies {
     }
     
     implementation group: 'org.apache.logging.log4j', name: 'log4j-api', version: versions.log4j
-    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: versions.log4j
+//    implementation group: 'org.apache.logging.log4j', name: 'log4j-core', version: versions.log4j
```

Comment:
Can you remove this instead of commenting?

Reply:
thanks, addressed here [bf5a2c8](https://github.com/hmcts/rd-user-profile-api/pull/813/commits/bf5a2c81407d6114bdfe032ef680114e86d7a78d)